README.md

```diff
@@ -221,3 +221,9 @@ There is [docker documentation](https://docs.docker.com/storage/volumes/#backup-
 * As mentioned above, there is a `.env` file containing application secrets. This **must not** be checked into version control!
 * The system is configured on start-up using the `docker-entrypoint.sh` script, which configures users in the `seed/setup.json` file.
 * Importers are run manually in the container using the rails console. See [The project wiki](https://github.com/antleaf/nims-hyrax/wiki) for more information.
+
+## Code Status
+
+[![Codeship Status for antleaf/nims-hyrax](https://app.codeship.com/projects/d4cc8560-e430-0136-fffd-6a7889452552/status?branch=master)](https://app.codeship.com/projects/319029)
```

Comment:
develop branch might be more relevant here, since we're actively developing there.

Reply:
@Steven-Eardley good catch, now tweaked